### PR TITLE
Created center the map on a clicked symbol feature; added 2 more symbols/icons on the map

### PR DESCRIPTION
**Pull request by tsarg88**
…ols/icons on the map

# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -20,8 +20,10 @@
     -->
     <link href='https://api.mapbox.com/mapbox-gl-js/v1.2.0/mapbox-gl.css' rel='stylesheet' />
     <link href='https://api.mapbox.com/mapbox-assembly/mbx/v0.18.0/assembly.min.css' rel='stylesheet'>
-    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.39.1/mapbox-gl.css' rel='stylesheet' />
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.4.0/mapbox-gl.css' rel='stylesheet' />
```
Comment (Bryce-Soghigian) [inline public/index.html:23]: We can't mess with that tileset

**Review by Bryce-Soghigian — COMMENTED** [review]
Why did you change all the single quotes to double quotes? what exactly did you change?

File: public/index.html (hunk 1 above)
Comment (tsarg88) [reply public/index.html:23]: I used prettier extension to indent the code, and most likely that is the reason of double quotes.

Comment (tsarg88) [reply public/index.html:23]: My updated tile is new one which is coming from the same mapbox, and I have tested the app to make sure that it is working.

**Pull request merged**